account_commission/models/account_move.py

```diff
@@ -136,6 +136,13 @@ def _compute_any_settled(self):
         for record in self:
             record.any_settled = any(record.mapped("agent_ids.settled"))
 
+    def filter_commission_applicable_lines(self):
```

Comment:
Make it private, as it's not called from RPC:

```suggestion
    def _filter_commission_applicable_lines(self):
```

Reply:
Done!